Src/DependencyCollector/Shared/Implementation/SqlClientDiagnostics/SqlClientDiagnosticSourceListener.cs

```diff
@@ -116,7 +118,14 @@ public void Dispose()
                                 Data = command.CommandText,
                                 Success = true
                             };
-                            
+
+                            // Populate the operation details for intializers
+                            telemetry.OperationDetails[RemoteDependencyConstants.SqlCommandOperationDetailName] = command;
+                            if (connection != null)
+                            {
+                                telemetry.OperationDetails[RemoteDependencyConstants.SqlConnectionOperationDetailName] = connection;
```

Comment:
The connection is available through the command, why do we need to explicitly add it to the dictionary?

Reply:
Similar rationale as the response\headers comment above.  I was not sure that the command would always be populated based on the failure cases, and I wanted to make it easier to develop the initializer if someone was only interested in properties on the connection.  I will wait for confirmation on the comment above, and align the two to be consistent.